src/Sylius/Bundle/ShopBundle/Form/Extension/ChannelTypeExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Form\Extension;
+
+use Sylius\Bundle\ThemeBundle\Form\Type\ThemeNameChoiceType;
+use Symfony\Component\Form\AbstractTypeExtension;
+use Sylius\Bundle\ChannelBundle\Form\Type\ChannelType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+final class ChannelTypeExtension extends AbstractTypeExtension
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        if (class_exists('\Sylius\Bundle\ThemeBundle\Form\Type\ThemeNameChoiceType')) {
```

Comment:
Is it possible to move fqcn to `use` and use it here as `ThemeNameChoiceType::class`?

Or would it be better to check for ThemeBundle presence instead and let it fail if for some reason `ThemeNameChoiceType` would be removed? Because otherwise you can have ThemeBundle and not this form type and code will continue to work correctly, which is not desired I think.

Reply:
It would probably make sense to check for the presence of the ThemeBundle, this way the code would notice if the `ThemeNameChoiceType` doesn't exist in the bundle anymore.